help/include/lists.md

```diff
@@ -0,0 +1,43 @@
+## Bulleted lists
+
+You can create bulleted lists using `*`, `-`, or `+` at the start of each line.
+Add two spaces before the bullet to create a nested list.
+
+### What you type
+
+```
+* bulleted lists
+  * with sub-bullets too
+  * sub-bullets start with 2 spaces
+    * start sub-sub-bullets with 4 spaces
+* multi
+line
+bullet
+- dashes and
++ pluses are ok too
+```
+
+### What it looks like
+
+![Markdown bullets](/static/images/help/markdown-bullets.png)
+
+## Numbered lists
+
+You can create numbered lists by putting a number followed by a `.` at the start
+of each line. Lists are numbered automatically, so you can reorder list items
+while editing your message without having to update the numbers. Add two spaces
+before the number to create a nested list.
```

Comment:
We should have an example that starts with a number above 1 and/or talk about how if you do that, the automatic numbering does the nice thing of counting from your starting number.

Reply:
Something like this, maybe?


```
1. numbered lists
1. increment automatically
   1. use nested lists if you like
   3. delete or reorder lines without fixing the numbering
1. one more
   101. lists can start at any number 
   102. so you can continue a list after some other text
```

Reply:
Yeah, that works for me! One could replace the `18.` with `1.`, or add a `1.` after it, if we want.